.github/workflows/debs.yml

```diff
@@ -0,0 +1,44 @@
+name: Debian Packaging CI
+on: [push, pull_request]
+
+jobs:
+  build:
+    runs-on: ${{ matrix.os }}
+    strategy:
+      fail-fast: false
+      matrix:
+        os: [ubuntu-20.04, ubuntu-22.04]
+    defaults:
+      run:
+        shell: bash
+
+    steps:
+    - name: Checkout code
+      uses: actions/checkout@v4
+      with:
+        ref: ${{ github.event.sha }}
+        fetch-depth: 0
+        fetch-tags: true
+    - name: Get tags from upstream, if needed
+      if: github.repository != 'Azure/azure-nvme-utils'
+      run: |
+        git remote add upstream https://github.com/Azure/azure-nvme-utils.git
+        git fetch upstream --tags
+    - name: Setup
+      run: |
+        sudo apt update
+        sudo apt install gcc pandoc cmake devscripts debhelper -y
+    - name: Build debs
+      run: |
+        ./scripts/build-deb.sh
+    - name: Install debs
```

Comment:
It'd be good to invoke `lintian` in a GitHub workflow step. We want to ensure we're compliant with distro policies and best practice.  It should be sufficient for now to install the `lintian` package and then invoke `lintian out/*.changes`

Reply:
added lintian check, but in my local build I'm seeing a warning during build I don't see when running lintian separately:

```
Now running lintian azure-nvme-utils_0.1.2+git393c9ca_amd64.changes ...
W: azure-nvme-utils-dbgsym: elf-error In program headers: Unable to find program interpreter name [usr/lib/debug/.build-id/b1/e816f72bb315c47eb8f759854aca258476017a.debug]
Finished running lintian.
```

Any idea?

Reply:
I'm not seeing this error in current builds in a sid environment, though I was when there were leftover files in packaging/debian/... from a previous build done when that was symlinked to the top-level debian subdirectory.  So I think this particular lintian alert is a non-issue.  However, there do seem to be some new ones...

Reply:
Addressing with https://github.com/Azure/azure-nvme-utils/pull/27